### ISSUE TITLE
[Feature Request] A few ideas for the UI

### ISSUE THREAD

**Issue by MagosCawl** [dr-nyt/NovelScraper#30]
Hi,

As requested (sorry for delay, computer died and just got back online!), here are the ideas I have for the app:

- A compact view for those of us who like to add lots of books
- Ability to differentiate between downloaded and yet to be downloaded novels (maybe a toggle, or a different section for each)
- An update icon for already downloaded novels instead of a download icon. 
- Currently, when updating a novel, it indicates that the download start from scratch so maybe the messaging in the progress bar could be changed

**Comment by dr-nyt** [comment]
`A compact view for those of us who like to add lots of books`
I could add a setting to enable a quick download button on top of the cover page.

`Ability to differentiate between downloaded and yet to be downloaded novel`
Currently, in the released Version 1.0.0 downloaded novels have a green bar around them to indicate they are downloaded. I might consider making different sections in the future.

`An update icon for already downloaded novels instead of a download icon.`
Already fixed in Version 1.0.0

`Currently, when updating a novel, it indicates that the download start from scratch so maybe the messaging in the progress bar could be changed`
I might consider this in the future, but currently, I think it is fine.

**Comment by Gweenteaneko** [comment]
yep he and I have the same concern. but UI is somewhat a little complicated for me

**Comment by dr-nyt** [comment]
> yep he and I have the same concern. but UI is somewhat a little complicated for me
How would you improve the UI if you could?

I was also thinking of having a different page where it lists all the queued downloads.

**Comment by Gweenteaneko** [comment]
uhmm. i think ditching the grid style would simplify everything and revert it to old style that way you can put like download progress, and many more
